app/navigation/Navigator.scala

```diff
@@ -73,7 +74,7 @@ class Navigator @Inject()() {
   }
 
   private def indirectRepresentativeRoute(answers: UserAnswers): Call = answers.get(IndirectRepresentativePage) match {
-    case Some(true)  => routes.BankDetailsController.onPageLoad(NormalMode)
+    case Some(true)  => routes.ProofOfAuthorityController.onPageLoad()
```

Comment:
I think this is wrong. If the user selects 'Yes' they should go to the BankDetails Page. If they select No they should go to the proofofauthority page. That's why I left the case(false) as not implemented (???).